controllers/dspipeline_params.go

```diff
@@ -578,6 +582,7 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 	p.APIServer = dsp.Spec.APIServer.DeepCopy()
 	p.APIServerDefaultResourceName = apiServerDefaultResourceNamePrefix + dsp.Name
 	p.APIServerServiceName = fmt.Sprintf("%s-%s", config.DSPServicePrefix, p.Name)
+	p.APIServerServiceDNSName = fmt.Sprintf("%s.%s.svc.cluster.local", p.APIServerServiceName, p.Namespace)
```

Comment:
hm, no kube API to build this string?

meh, it's probably ok to hardcode `svc.cluster.local`

Reply:
if you mean kube api server, then I'd prefer this method instead of querying the api for what is a fairly static configuration

if you mean some package utility that would do it for us, then I don't know of one

Reply:
> if you mean some package utility that would do it for us, then I don't know of one

that is what I meant.